tests/test_imsim_cmpt.py

```diff
@@ -197,6 +197,7 @@ def test_analyze_opd_data(self):
         ans_zk = get_zk_from_file(ans_zk_file_path)
 
         for det in [191, 195, 199, 203]:
+            print(zk[det], ans_zk[det])
```

Comment:
If you could remove this print line, that would be great! Just a small comment that you are probably aware of.

Reply:
I missed that. Thanks for finding it!